client/container.go

```diff
@@ -732,19 +612,21 @@ func (x *PrmAnnounceSpace) SetValues(vs []container.SizeEstimation) {
 //
 // Context is required and must not be nil. It is used for network communication.
 //
+// Announcements parameter MUST NOT be empty slice.
```

Comment:
i mean, i would not add that line or would not be so strict (SHOULD instead of MUST)

Reply:
It makes, sense. Changed to `should`

Reply:
just in case: as i understand, we usually take in mind [that rfc](https://www.ietf.org/rfc/rfc2119.txt) when choosing wording

cc @roman-khimov, @cthulhu-rider

Reply:
Yes. In this case an empty slice leads to an immediate error, so likely we don't have a case of `there may exist valid reasons in particular circumstances to ignore a particular item`.